crates/autopilot/src/run_loop.rs

```diff
@@ -307,12 +309,16 @@ impl RunLoop {
         id: domain::auction::Id,
         auction: &domain::Auction,
     ) -> Vec<Participant<'_>> {
+        let mut surplus_capturing_jit_order_owners =
+            self.cow_amm_indexer.get_cow_amm_addresses().await;
+        surplus_capturing_jit_order_owners.extend(self.surplus_capturing_jit_order_owners.clone());
```

Comment:
should we also add these addresses to the protocol fee exempt addresses?

Reply:
Yes. Technically this will very likely not be needed once the watchtower stop indexing these template orders but even then edge cases exist where this is needed.